composer.json

```diff
@@ -17,7 +17,7 @@
 	],
 	"require": {
 		"php": ">=7.1",
-		"nette/di": "~2.4.14",
+		"nette/di": "~2.4.14 || ^3.0",
```

Comment:
I would prefer `~3.0.0`, because Nette does not follow strictly semver. :-)

Reply:
Fixed :-)